docker-entrypoint.sh

```diff
@@ -62,30 +62,40 @@ if expr "$*" : 'java .*/start\.jar.*$' >/dev/null ; then
 		esac
 	done
 
-	if [ -f /jetty-start ] ; then
-		if [ $JETTY_BASE/start.d -nt /jetty-start ] ; then
+	if [ $(whoami) != "jetty" ]; then
+		cat >&2 <<- EOWARN
+			********************************************************************
+			WARNING: User is $(whoami)
+			         The user should be (re)set to 'jetty' in the Dockerfile
+			********************************************************************
+		EOWARN
+	fi
+
```

Comment:
I think it would make sense to add the following here:

```shell
: ${JETTY_START:=$JETTY_BASE/jetty.start}
```

Then, all the places below that refer to `$JETTY_BASE/jetty.start` should be changed to `"$JETTY_START"`.

This allows a user to pass in `-e JETTY_START` to specify an alternate location that could be located on a volume or `tmpfs`.

Reply:
Good idea, but I'll set it the old fashioned way for `sh` in alpine.

Reply:
BTW, the syntax in my previous comment is POSIX syntax and works in Alpine's `/bin/sh` as well. See http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02

Reply:
The colon is also a POSIX "special built-in": http://pubs.opengroup.org/onlinepubs/009695399/utilities/colon.html

Reply:
Oh really,  so I'm the really old fashioned one here.

Happy to update to the more concise syntax, but probably should do it for other variables

Reply:
update to posix syntax